cylc/flow/scheduler.py

```diff
@@ -678,17 +678,15 @@ def _load_pool_from_point(self):
         released from runhead.)
 
         """
-        if self.config.start_point is not None:
-            start_type = "Warm" if self.options.startcp else "Cold"
-            LOG.info(f"{start_type} start from {self.config.start_point}")
+        start_type = "Cold"
+        if self.config.start_point > self.config.initial_point:
+            start_type = "Warm"
+        LOG.info(f"{start_type} start from {self.config.start_point}")
```

Comment:
This is definitley more readable.

One more alternative?
```suggestion
LOG.info(
    f"{'warm' if self.options.startcp else 'cold'} start from {self.config.start_point}.")
```

Reply:
I'll take this and merge :grin: